markdown/usage/troubleshooting.md

```diff
@@ -3,36 +3,98 @@ title: Troubleshooting
 subtitle: How to troubleshoot common mistakes and issues
 ---
 
-* [Input files not found](#input-files-not-found)
-* [Data organization](#data-organization)
-* [Image cannot be built](#image-cannot-be-built)
-* [Cannot find input files when using Singularity](#cannot-find-input-files-when-using-singularity)
-* [Warning about 'sticked on revision'](#warning-about-sticked-on-revision)
-* [Extra resources and getting help](#extra-resources-and-getting-help)
+- [Input files not found](#input-files-not-found)
+  - [Direct input](#direct-input)
+  - [Sample sheet input](#sample-sheet-input)
+  - [Output for only a single sample although I specified multiple with wildcards](#output-for-only-a-single-sample-although-i-specified-multiple-with-wildcards)
+  - [Data organization](#data-organization)
+- [The pipeline crashes almost immediately with an early pipeline step](#the-pipeline-crashes-almost-immediately-with-an-early-pipeline-step)
+  - [Error related to Docker](#error-related-to-docker)
+  - [Error related to Singularity](#error-related-to-singularity)
+- [Cannot find input files when using Singularity](#cannot-find-input-files-when-using-singularity)
+- [Warning about sticked on revision](#warning-about-sticked-on-revision)
+- [I get a exceeded job memory limit error](#i-get-a-exceeded-job-memory-limit-error)
+- [Crashed pipeline with an error but Nextflow is still running](#crashed-pipeline-with-an-error-but-nextflow-is-still-running)
+  - [A step of a pipeline wasn't executed](#a-step-of-a-pipeline-wasnt-executed)
+- [My pipeline update doesn't seem to do anything](#my-pipeline-update-doesnt-seem-to-do-anything)
+- [Unable to acquire lock error](#unable-to-acquire-lock-error)
+- [Extra resources and getting help](#extra-resources-and-getting-help)
 
 ## Input files not found
 
-If only no file, only one input file , or only read one and not read two is picked up then something is wrong with your input file declaration
+If the pipeline can't find your files then you will get the following error
+
+```bash
+ERROR ~ Cannot find any reads matching: *{1,2}.fastq.gz
```

Comment:
Could we have a minimal example that lead to this error?
I think error could be more understandable if we can see what is happening there.

I'm guessing here, the problem is most likely a typo